tests/README.md

```diff
@@ -163,125 +313,22 @@ Example of a scenario:
 echo "KerberosAuthentication yes" >> /etc/ssh/sshd_config
 ```
 
-After the header arbitrary bash commands follow.
+After the header bash commands to prepare the scenario follow.
```

Comment:
Strange sentence, I would say "Bash commands to prepare the scenario follow after the header."